dev.rdm.RDM.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
Consider making it part of your code repository as it's not specific to Flathub.